modules/rhel-adding-node.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * machine_management/adding-rhel-compute.adoc
+
+[id="rhel-adding-node-{context}"]
+= Adding a RHEL compute machine to your cluster
+
+You can add compute machines that use Red Hat Enterprise Linux as the operating
+system to a {product-title} {product-version} cluster.
+
+.Prerequisites
+
+* The machine that you run the playbook on must be able to access all of the
+RHEL hosts. You can use any method that your company allows, including a
+bastion with an SSH proxy or a VPN.
+* The `kubeconfig` file for the cluster and the installation program that you
+used to install the cluster are on the machine that you use the run the playbook.
+* You must prepare the RHEL hosts for installation.
+* Configure a user that has SSH access to all of the RHEL hosts.
+* If you use SSH key-based authentication, you must manage the key with an
+SSH agent.
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+* Install the `jq` package.
+
```

Comment:
Add "Install Ansible 2.7 on the control host"
Add "Install OpenShift-Ansible on the control host"

Reply:
There are so many things that you need to do to the control host that I'm making a new module about configuring it.